controllers/record_match_run.go

```diff
@@ -96,7 +103,7 @@ func CreateRecordMatchRunHandler(provider func() *mgo.Database) gin.HandlerFunc
 				"server endpoint": svrEndpoint,
 				"reqBody":         string(reqBody[:]),
 				"message":         reqMatchRequest.Message,
-				"request":         reqMatchRequest}).Info("About to submit request")
+				"request":         string(reqBody)}).Info("About to submit request")
```

Comment:
Glad you caught it.  I was looking right past it even after making the change so the address of reqMatchRequest wasn't printed out.